src/sdkV2.ts

```diff
@@ -2300,6 +2272,24 @@ export default class CegaEvmSDKV2 {
     });
   }
 
+  async sfnCheckTradeExpiry(
```

Comment:
we already have a bulkCheckTradeExpiry, don't we?

Reply:
@bonnopc please check again

I see this on main:
```
  async sfnBulkCheckTradesExpiry(
    vaultAddresses: EvmAddress[],
    overrides: TxOverrides = {},
  ): Promise<ethers.providers.TransactionResponse> {
    const cegaEntry = await this.loadCegaEntry();

    return cegaEntry.sfnBulkCheckTradesExpiry(vaultAddresses, {
      ...(await this._gasStation.getGasOraclePrices()),
      ...(await getOverridesWithEstimatedGasLimit(
        cegaEntry,
        'sfnBulkCheckTradesExpiry',
        [vaultAddresses],
        this._signer,
        overrides,
      )),
    });
  }
```

Reply:
@vsai Sorry, :cold_sweat: My bad, was using "CheckTradeExpiry" to search it before! Now removed it